### ISSUE TITLE
Share hashtag feed externally

### ISSUE THREAD

**Issue by alltheseas** [damus-io/notecrumbs#15]
See

https://bsky.app/hashtag/Cats

![image](https://github.com/user-attachments/assets/d0799ebb-c8e1-4986-8dda-17d23c7bc514)

**Comment by alltheseas** [comment]
@dergigi wants this 

https://damus.io/nevent1qqsggzs7zqqrgea3a5ezdjjaqrhpvp4tfg8afdshyhmwvwa85aq52asa9s55k

> I often find myself trying to link to a nostr hashtag, just like I often want to link to an npub in a neutral way. For the npub it's easy, but for a hashtag I have to resort to nostr band or similar, which isn't really a neutral or generic link. Are there any efforts underway to implement this? Or related: linking to a search.

**Comment by dergigi** [comment]
Yes! But just to clarify, what I'm thinking of is a protocol-native link like we have for npubs: `nostr:npub1dergggklka99wwrs92yz8wdjs952h2ux2ha2ed598ngwu9w7a6fsh9xzpc`

Not sure if this is spec'd out or not, but I'd imagine it would be something like `nostr:#cats`

**Comment by alltheseas** [comment]
Might we add hashtags to NIP-21 

@jb55 @fiatjaf ?

For instance

`nostr:#cats`

**Comment by jb55** [comment]
damus handles `nostr:t:cats`

**Comment by fiatjaf** [comment]
I think this has very limited value unless you also specify the relays you want to read from.

**Comment by jb55** [comment]
nostr:t:cats?r=relay.damus.io&r=pyramid.fiatjaf.com
